scripts/postinstall.js

```diff
@@ -30,8 +30,48 @@ const removeUnwantedFolders = async (root, unwantedNames) => {
 
   return promises;
 };
+
+const patchFile = async (file, patch) => {
+  console.log(`Patching ${file}`);
+  const patches = Array.isArray(patch) ? patch : [patch];
+  let fileContent = await fs.readFile(file, 'utf8');
+  for (const { from, to } of patches) {
+    // The splitting by `to` is to make sure we don't patch already patched ones
+    fileContent = fileContent
+      .split(to)
+      .map((token) => token.split(from))
+      .flat()
+      .join(to);
+  }
+  await fs.writeFile(file, fileContent);
+};
+
 const run = async () => {
   const promises = await removeUnwantedFolders('node_modules', ['demo', 'example', 'examples']);
+
+  promises.push(
+    patchFile('node_modules/font-awesome/scss/_variables.scss', {
+      from: '(30em / 14)',
```

Comment:
why don't we use `calc` here?

Reply:
FontAwesome doesn't use `calc` and that's why we are patching it here from `(30em / 14)` to `calc(30em / 14)`.

Reply:
Got it. Thanks for sharing information.